ocrd_modelfactory/ocrd_modelfactory/__init__.py

```diff
@@ -53,7 +56,7 @@ def page_from_image(input_file):
             imageWidth=exif.width,
             imageHeight=exif.height,
             # XXX brittle
```

Comment:
Still brittle?

Reply:
Yeah, fix is coming in another branch to solve the problem of imageFilename once and hopefully for all.